src/Select/Select.tsx

```diff
@@ -109,25 +119,31 @@ const InnerSelect = React.forwardRef<HTMLDivElement, SelectInnerProps>(
                 <Line />
               </ResetIconContainer>
             )}
-            <ElementRightContainer>
+            <ElementRightContainer onMouseOver={(e) => {
+              e.stopPropagation()
```

Comment:
Why stopping propagation ?

Reply:
Because I didn't want it to open `Select` using `onMouseOver` from parent.